### PR TITLE
feat: Add method for Has Content warning feature

### PR DESCRIPTION
**Pull request by ataromoku**
### Related Rally Story(s):
[US146577](https://rally1.rallydev.com/#/?detail=/userstory/673475675019&fdp=true): Course Merge - UI - Display course has content and/or user progress tags in course list

### Related PR(s):
* https://github.com/Brightspace/ipsis-sis-course-merge/pull/83

### CODE REVIEW

**Review by zwilson2 — APPROVED** [review]

File: src/ipsis/course-merge/CourseMergeOfferingEntity.js [hunk 1]
```diff
@@ -67,6 +67,10 @@ export class CourseMergeOfferingEntity extends Entity {
 		return this._entity?.hasActionByName(Actions.ipsis.sisCourseMerge.selectAsTarget);
 	}
 
+	hasContent() {
+		return this._entity?.properties?.hasContent;
+	}
```
Comment (JeremyWinkler) [inline src/ipsis/course-merge/CourseMergeOfferingEntity.js:72]: Nit: could we move this up to where the other properties are?

**Review by JeremyWinkler — APPROVED** [review]
Approving anyways, but would like to see the change. 
Also, in the future can you request review from `triforce-og` and `triforce-og-core-reviewers` (or `triforce-og-core-offstack-reviewers` for non LMS repos). Those teams should auto-add people for review.

File: src/ipsis/course-merge/CourseMergeOfferingEntity.js (hunk 1 above)
Comment (ataromoku) [reply src/ipsis/course-merge/CourseMergeOfferingEntity.js]: Good idea, done.

Comment (ataromoku) [reply src/ipsis/course-merge/CourseMergeOfferingEntity.js]: What do you mean by `"but would like to see the change."`

**Pull request merged**